chapter-01-polynomes.tex

```diff
@@ -433,21 +433,21 @@ \section{Divisibilité et racines}
 \end{displaymath}
 tel que $\deg(r(x)) < \deg(g(x))$. On  obtient alors
 \begin{displaymath}
-  f(x) = \left(q(x) + \frac{a_n}{ b_m } x^{n-m} \right) g(x) + r(x). 
+  f(x) = \left(q(x) + \frac{a_n}{ b_m } x^{n-m} \right) g(x) + r(x) = q_0(x)g(x) + r(x), \;\text{où} q_0(x) = q(x) + \frac{a_n}{ b_m } x^{n-m}. 
```

Comment:
I am not very happy with this change. The quotient and remainder that are obtained by recurrence could be called $q'$ and $r'$ respectively. It is always better to stick to the names in the theorem.

Reply:
Do you mean that my $q_0(x)$ should be called $q'(x)$ instead? But later there's a $q'(x)$ already (maybe call it $q''(x)$ then? might be a little cluttered though).

Reply:
Please remove these particular changes in question. I will re-write the text and then you can see better what I mean.

Reply:
Should I still do something (i.e. remove the changes)? I'm a novice at git and it looks like you merged the changes you wanted.